assets/js/googlesitekit/widgets/register-defaults.js

```diff
@@ -87,8 +87,8 @@ export function registerDefaults( widgetsAPI ) {
 		// Main dashboard
 		AREA_MAIN_DASHBOARD_KEY_METRICS_PRIMARY,
 		AREA_MAIN_DASHBOARD_TRAFFIC_PRIMARY,
-		AREA_MAIN_DASHBOARD_TRAFFIC_AUDIENCE_SEGMENTATION,
 		AREA_MAIN_DASHBOARD_CONTENT_PRIMARY,
+		AREA_MAIN_DASHBOARD_TRAFFIC_AUDIENCE_SEGMENTATION,
```

Comment:
Looks like this was moved out of order by accident, I'll revert the change.

```suggestion
		AREA_MAIN_DASHBOARD_TRAFFIC_PRIMARY,
		AREA_MAIN_DASHBOARD_TRAFFIC_AUDIENCE_SEGMENTATION,
```